### ISSUE TITLE
Decomposing Leather Helmet crashes the game

### ISSUE THREAD

**Issue by Unrepentant-Atheist** [AdlyTempleton/MineChem#155]
Crash log: http://paste.ee/p/zxnNW

**Comment by jakimfett** [comment]
I tested this a couple of times, but was unable to reproduce the issue. Couple of questions for you about the crash:
1. At what point did the crash happen? 
2. Can you tell me what version of Minechem you were using? 
3. What power source did the Decomposer have? 
4. Was it a single item that you put in to decompose? 
5. Did you put it in manually or via a Buildcraft Pipe/Thermal Expansion Itemduct?
6. Was the decomposer full, or was there space for the three types of material?

Basically, the more info you give me, the easier it is for me to track this down. I'm going to go ahead and close this issue, but if I get a response with more details, I'll re-open it.
